xml/gl.xml

```diff
@@ -2795,10 +2795,10 @@ typedef unsigned int GLhandleARB;
         <enum value="0x8512" name="GL_REFLECTION_MAP_EXT"/>
         <enum value="0x8512" name="GL_REFLECTION_MAP_NV"/>
         <enum value="0x8512" name="GL_REFLECTION_MAP_OES"/>
-        <enum value="0x8513" name="GL_TEXTURE_CUBE_MAP" group="CopyImageSubDataTarget,TextureTarget"/>
-        <enum value="0x8513" name="GL_TEXTURE_CUBE_MAP_ARB"/>
-        <enum value="0x8513" name="GL_TEXTURE_CUBE_MAP_EXT"/>
-        <enum value="0x8513" name="GL_TEXTURE_CUBE_MAP_OES"/>
+        <enum value="0x8513" name="GL_TEXTURE_CUBE_MAP" group="CopyImageSubDataTarget,TextureTarget,EnableCap,GetPName"/>
+        <enum value="0x8513" name="GL_TEXTURE_CUBE_MAP_ARB" group="TextureTarget,EnableCap,GetPName"/>
+        <enum value="0x8513" name="GL_TEXTURE_CUBE_MAP_EXT" group="TextureTarget,EnableCap,GetPName"/>
+        <enum value="0x8513" name="GL_TEXTURE_CUBE_MAP_OES" group="TextureTarget,EnableCap,GetPName"/>
```

Comment:
Why add `GetPName` for these?
`glGet` takes `GL_TEXTURE_BINDING_CUBE_MAP` and not `GL_TEXTURE_CUBE_MAP`.

Reply:
From ARB_texture_cube_map (included in OpenGL 3.1)
    _Accepted by the <cap> parameter of Enable, Disable, IsEnabled, and
    by the <pname> parameter of GetBooleanv, GetIntegerv, GetFloatv,
    and GetDoublev, and by the <target> parameter of BindTexture,
    GetTexParameterfv, GetTexParameteriv, TexParameterf, TexParameteri,
    TexParameterfv, and TexParameteriv:

        TEXTURE_CUBE_MAP_ARB                0x8513_